README.md

```diff
@@ -41,18 +41,180 @@ action:
 
 ```
 
-***NOTE:*** For database action, remember to set up your Aito instance credentials.
+***NOTE:*** For the database action, remember to set up your Aito instance credentials.
 
-For addition guide of the CLI tool, see the [CLI documentations](docs/cli.md)
+For an additional guide of the CLI tool, see the [CLI documentations](docs/cli.md)
 
-### Integrating with [pandas](https://pandas.pydata.org/) DataFrame
+### Getting your data into Aito using Python
 
-* Infer Aito Schema from a pandas DataFrame:
+To get you data into Aito you will have to go through the following steps:
+
+1. [Define Aito schema](#1-define-aito-schema)
+2. [Upload Aito schema](#2-upload-aito-schema)
+3. [Upload data](#3-upload-data)
+
+#### 1. Define Aito schema 
+
+You can either create the schema manually, by using your own code or use the infer feature of the Aito python library.
+
+You can use [pandas](https://pandas.pydata.org/) DataFrame to infer your Aito schema.
+
+For more information on the Aito schema can be found [here](https://aito.ai/docs/articles/defining-a-database-schema/)
+
+##### Infer Aito Schema from a pandas DataFrame
   ```python
+  import pandas as pd
   from aito.utils.schema_handler import SchemaHandler
 
+  # Load a CSV into a pandas Dataframe
+  dataframe = pd.read_csv(your_file_path, delimiter=";")
```

Comment:
Could we always give users and example file that these instructions are guaranteed to work with? I like that a lot with some other examples of similiar products.